cfme/utils/net.py

```diff
@@ -165,15 +165,16 @@ def find_pingable(mgmt_vm, allow_ipv6=True):
          In priority: first pingable address, address 'selected' by wrapanapi (possibly None)
      """
     for ip in getattr(mgmt_vm, 'all_ips', []):
-        if not allow_ipv6 and is_ipv6(ip):
-            logger.debug('VMs ip is ipv6, skipping it: %s', ip)
-            continue
-        if not is_pingable(ip):
-            logger.debug('Could not reach mgmt IP on VM: %s', ip)
-            continue
-
-        logger.info('Found reachable IP for VM: %s', ip)
-        return ip
+        if ip is not None:
```

Comment:
What is the reason this change is necessary? 
```suggestion
        if ip:
```

Reply:
Or could just have 
```python
            if not ip:
                logger.debug('blah blah blah')
                continue
            if not allow_ipv6 and is_ipv6(ip):
                logger.debug('VMs ip is ipv6, skipping it: %s', ip)
                continue
            if not is_pingable(ip):
                logger.debug('Could not reach mgmt IP on VM: %s', ip)
                continue
```

Reply:
@john-dupuy so the reason I had to add that enhancement was because in some cases it was failing when `ip` would be `None`. 
Hence I added the check. But iirc I don't use this function anymore in this PR. That being said, I think your first suggestion is more valid, cause if I use second comment suggestion above, `is_ipv6(ip)` may fail when `ip=None`. Hence checking `ip` to be `non-None` should be wrapper check. Hope that makes sense.